.coveragerc

```diff
@@ -1,6 +1,6 @@
 [run]
-source = eplus_rmd
+source = energyplus_rmd
```

Comment:
Almost all changes here are just renaming the files and using the new source folder name.

Reply:
If any users are accessing this package as a library, they will just need to update their `import` statements.  Pretty minor.